.changeset/pre.json

```diff
@@ -69,7 +69,7 @@
     "eslint-config-vercel-ai": "0.0.0",
     "generate-llms-txt": "0.0.0",
     "@vercel/ai-tsconfig": "0.0.0",
-    "@ai-sdk/baseten": "0.0.0",
+    "@ai-sdk/baseten": "0.0.1",
```

Comment:
I think we might not even need it, but also it won't harm. It just reflects what we would have had if we did the release through `release-v5.0` branch first, then ported that to `main` instead of the other way around.